game-headed/build.install4j

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<install4j version="7.0.8" transformSequenceNumber="7">
+<install4j version="7.0.9" transformSequenceNumber="7">
```

Comment:
When I load this file in the 7.0.9 IDE and save it without any changes, this is the only line that is modified.  Are the changes below intentional?

Reply:
Yes, they are the result of myself doing the same action

Reply:
No known benefit to changing these lines, easy enough to keep them as is: b282b6836